Code/GraphMol/CIPLabeler/CMakeLists.txt

```diff
@@ -0,0 +1,35 @@
+
+remove_definitions(-DRDKIT_GRAPHMOL_BUILD)
+add_definitions(-DRDKIT_CIPLABELER_BUILD)
+
+file(GLOB configs_src "configs/*.cpp")
```

Comment:
the "effective modern cmake" guidelines advise against using `file(GLOB)`:
https://gist.github.com/mbinna/c61dbb39bca0e4fb7d1f73b0d66a4fd1#dont-use-fileglob-in-projects

The argument against it seems like a good one.

Reply:
Ok, done

Reply:
nice reference, thanks.